EasyPost/Batch.cs

```diff
@@ -1,134 +1,157 @@
-﻿using RestSharp;
+﻿// Batch.cs
+// Copyright (c) 2022 EasyPost
+// All rights reserved.
 
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using RestSharp;
 
-namespace EasyPost {
-    public class Batch : Resource {
-#pragma warning disable IDE1006 // Naming Styles
```

Comment:
Had to remove the `#pragma` annotations to get the auto-grouping functionality to work.
The `pragma` annotation is just to get your Visual Studio to stop complaining that the variables don't follow the expected naming style. It has no impact on the functionality or compilation of the code (and fun fact, that warning doesn't work in Rider, I had to use a different `pragma` annotation on my end - since removed).

Reply:
Does Visual Studio now throw a bunch of errors because we no longer disable this warning?

Reply:
It's not on my end.

Reply:
Cool, thanks!